Lib/test/inspect_fodder2.py

```diff
@@ -187,24 +187,51 @@ class cls185:
         def func186(self):
             pass
 
-
 def class_decorator(cls):
     return cls
 
-# line 195
+# line 193
 @class_decorator
 @class_decorator
-class cls197:
+class cls196:
 
     @class_decorator
     @class_decorator
-    class cls201:
+    class cls200:
         pass
 
-class cls204:
-    class cls205:
-        class cls206:
+class cls203:
+    class cls204:
+        class cls205:
+            pass
+    class cls207:
+        class cls208:
```

Comment:
Maybe use `cls205` for testing classes with same name but different `__qualname__`?

Reply:
Used `cls205` in the test. Thanks.